net/instaweb/rewriter/public/rewrite_options.h

```diff
@@ -1527,6 +1528,13 @@ class RewriteOptions {
     set_option(x, &blink_blacklist_end_timestamp_ms_);
   }
 
+  bool preserve_subresource_hints() const {
```

Comment:
I like the phrasing as you have it: when we (eventually) extend PageSpeed to support inserting new subresource hints the `PreserveSubresourceHints` will still mean "keep my existing hints as they are" while `StripSubresourceHints` won't be as clear.